tests/ui/feed.js

```diff
@@ -32,7 +32,7 @@ var feedDetailPages = [
         path: '/feed/collection/coll-promo',
     },
     {
-        name: 'Collection Promo Desc',
+        name: 'Collection Promo &amp; Desc',
```

Comment:
this won't actually change the title of the collection so the check below would be moot